inc/front/minify.php

```diff
@@ -314,6 +387,70 @@ function rocket_minify_css( $buffer )
 }
 
 /**
+ * Minify JS files without concatenation.
+ *
+ * @since 3.0
+ * @author Remy Perona
+ *
+ * @param string $buffer HTML code to parse.
+ * @return string Updated HTML code
+ */
+function rocket_minify_js_only( $buffer ) {
+	// Get host of CNAMES.
```

Comment:
Could we create a function for the lines from 399 to 413. We use exactly the same code for the CSS part. With an argument (css or js in value) to use the right exclusion function.